config/initializers/foreman.rb

```diff
@@ -32,17 +31,17 @@
   Foreman.settings.load_values
 end
 
-# load topbar
-Menu::Loader.load
-
-# clear our users topbar cache
-# The users table may not be exist during initial migration of the database
-TopbarSweeper.expire_cache_all_users if (User.table_exists? rescue false)
-
 Foreman::Plugin.initialize_default_registries
 Foreman::Plugin.medium_providers_registry.register MediumProviders::Default
 
 Rails.application.config.to_prepare do
+  # load topbar
+  Menu::Loader.load
+
+  # clear our users topbar cache
+  # The users table may not be exist during initial migration of the database
+  TopbarSweeper.expire_cache_all_users if (User.table_exists? rescue false)
```

Comment:
Probably won't hurt to delete the cache on every code reload :shrug:
So I guess :+1: on this